### PR TITLE
networkctl: Add support to display bond

### PR DESCRIPTION
**Pull request by ssahani**
cc @yuwata

### CODE REVIEW

File: src/network/networkctl.c [hunk 1]
```diff
@@ -1405,6 +1415,35 @@ static int link_status_one(
...
+                        const char *mode;
+                } mode_table[] = {
+                        { "balance-rr" },
+                        { "active-backup" },
+                        { "balance-xor" },
+                        { "broadcast" },
+                        { "802.3ad" },
+                        { "balance-tlb" },
+                        { "balance-alb" },
+                };
```
Comment (yuwata) [inline src/network/networkctl.c:1429]: Maybe, create shared/bond-util.[ch] and expose `bond_mode_{to,from}_string()` in netdev/bond.c, then use it here.

Comment (ssahani) [reply src/network/networkctl.c:1429]: I thought of that wont's that be an overkill ?

Comment (yuwata) [reply src/network/networkctl.c:1429]: I thought then the functions can be used in network-generator. But OK. Let's merge this as is, and do that in the future.

Comment (ssahani) [reply src/network/networkctl.c:1429]: Then it would be useful. sure.

**Pull request merged**